google/ggrc-core
# GGRC-5462 Add paging for comments

# Issue description

Control (or any other model) with 2000 comments opens about 40 seconds.

# Steps to test the changes

1. Open any object
2. Add more then 10 comments

**Expected result:** paging appear

# Solution description

Add `comments-paging` component.

# Sanity checklist




# PR Review checklist
